### PR TITLE
Oauth

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
Pls review

### CODE REVIEW

File: src/components/WebsiteFooter/WebsiteFooter.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class WebsiteFooter extends Component {
                   <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/about"> About Us </Link> </li>
                   <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/terms-and-condtions"> Terms &amp; Conditions </Link></li>
                   <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/privacy-policy"> Privacy Policy </Link></li>
-                  <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/affiliatepage"> Affiliates </Link></li>
+                  {/* <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/affiliatepage"> Affiliates </Link></li> */}
```
Comment (ShankyRana) [inline src/components/WebsiteFooter/WebsiteFooter.js]: remove comment

File: src/components/AddChannel/AddChannel.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
+.website-app {
```
Comment (ShankyRana) [inline src/components/AddChannel/AddChannel.scss]: scss container should be of the component it is used for.

File: src/components/DashboardChannel/DashboardChannel.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    super();
+    this.state = {
+      selectedChannels: [],
+      channelContent: [],
+      checked: false
+    };
+  }
+
+  componentWillMount() {
+    window.scrollTo(0,0);
```
Comment (ShankyRana) [inline src/components/DashboardChannel/DashboardChannel.js]: this won't work with test cases, get something better than this related to react.

File: src/components/NotificationSettings/NotificationList/NotificationList.scss [hunk 4]
```diff
@@ -1,20 +1,38 @@
 @import '../../SwitchPartialScss/toggle';
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationList/NotificationList.scss]: is this being used globally?

File: src/components/Oauthgenerate/Oauthgenerate.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
+import React , { Component } from 'react';
+import './Oauthgenerate.scss';
+import { Row,Col } from 'react-bootstrap';
+// import { Link } from 'react-router';
```
Comment (ShankyRana) [inline src/components/Oauthgenerate/Oauthgenerate.js]: comment remove

File: src/components/PopupReview/PopupReview.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+import React , { Component } from 'react';
+import './PopupReview.scss';
+import { Row, Col } from 'react-bootstrap';
+// import { Link } from 'react-router';
```
Comment (ShankyRana) [inline src/components/PopupReview/PopupReview.js]: remove

File: src/components/PopupReview/PopupReview.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
+class PopupReview extends Component {
+  constructor() {
+    super();
+    this.state = {
```
Comment (ShankyRana) [inline src/components/PopupReview/PopupReview.js]: remove

File: src/components/WebsiteFooter/WebsiteFooter.js [hunk 8]
```diff
@@ -16,7 +16,7 @@ class WebsiteFooter extends Component {
                   <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/about"> About Us </Link> </li>
                   <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/terms-and-condtions"> Terms &amp; Conditions </Link></li>
                   <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/privacy-policy"> Privacy Policy </Link></li>
-                  <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/addchannel"> POP UP </Link></li>
+                  {/* <li className="type--uppercase col-sm-4 col-md-3 h6"><Link className="footer-menu-link" to="/addchannel"> POP UP </Link></li> */}
```
Comment (ShankyRana) [inline src/components/WebsiteFooter/WebsiteFooter.js]: remove

File: src/components/Oauthgenerate/Oauthgenerate.js [hunk 9]
```diff
@@ -56,11 +62,14 @@ class Oauthpage extends Component {
                     bsClass="form-control ml-3"
                     id="clientname"
                     placeholder="example: Ray-101, John doe"
-                    onChange={()=>{}}
+                    onChange={(e)=>{this.handleClientNameChange(e);}}
                     onBlur={()=>{}}
```
Comment (ShankyRana) [inline src/components/Oauthgenerate/Oauthgenerate.js]: remove

File: src/components/NotificationSettings/NotificationList/NotificationList.scss (hunk 4 above)
Comment (GAUTAMRAJU15) [reply src/components/NotificationSettings/NotificationList/NotificationList.scss]: this toggle button is global for both website and dashboard .,,the website has the pricing page using this toggle and dashboard too

Comment (ShankyRana) [reply src/components/NotificationSettings/NotificationList/NotificationList.scss]: so add this to a main file.

**Comment by Devilla** [comment]
Updated PR.

**Comment by Devilla** [comment]
Updated PR.

File: src/components/AddChannel/AddChannel.scss (hunk 2 above)
Comment (GAUTAMRAJU15) [reply src/components/AddChannel/AddChannel.scss]: okay sir

File: src/components/DashboardChannel/DashboardChannel.js (hunk 3 above)
Comment (GAUTAMRAJU15) [reply src/components/DashboardChannel/DashboardChannel.js]: var scrollElm = document.scrollingElement;
    scrollElm.scrollTop = 0;
this scrolls it

Comment (GAUTAMRAJU15) [reply src/components/DashboardChannel/DashboardChannel.js]: will this be right ?

**Pull request closed without merging**